cmd/frostfs-node/config.go

```diff
@@ -968,3 +968,36 @@ func (c *cfg) configWatcher(ctx context.Context) {
 		}
 	}
 }
+
+func (c *cfg) terminateWatcher(ctx context.Context) {
```

Comment:
Why did you decide to have a separate function?

Reply:
Now I see that it is not necessary, thanks. Merged two functions in one.